packages/core/src/properties/node.ts

```diff
@@ -176,29 +187,50 @@ export class Node extends ExtensibleProperty<INode> {
 	 * Scene hierarchy.
 	 */
 
-	/** Adds another node as a child of this one. Nodes cannot have multiple parents. */
+	/**
+	 * Adds the given Node as a child of this Node.
+	 *
+	 * Requirements:
+	 *
+	 * 1. Nodes MAY be root children of multiple {@link Scene Scenes}
+	 * 2. Nodes MUST NOT be children of >1 Node
+	 * 3. Nodes MUST NOT be children of both Nodes and {@link Scene Scenes}
+	 *
+	 * The `addChild` method enforces these restrictions automatically, and will
+	 * remove the new child from previous parents where needed. This behavior
+	 * may change in future major releases of the library.
+	 *
+	 * @privateRemarks Requires non-graph state.
+	 */
 	public addChild(child: Node): this {
-		// Remove existing parent.
+		// Remove existing parents.
 		if (child._parentNode) child._parentNode.removeChild(child);
+		if (child._parentScenes.size) {
+			const prevSize = child._parentScenes.size;
+			for (const scene of child._parentScenes) {
+				scene.removeChild(child);
+			}
+			console.log(`parentScenes: ${prevSize} -> ${child._parentScenes.size}`);
```

Comment:
Removed logging in https://github.com/donmccurdy/glTF-Transform/commit/2cfb488ae5ef76d370867e52f2403fc3d4e4e2ac.